### DISCUSSION TITLE
Help with Bootloader Updating

### DISCUSSION THREAD

**Discussion by buzzkillg230rc** [df8oe/UHSDR#1937] [Q&A] [answered]
HI,
I have a UHSDR QRP on Bootloader 3.4.2. I was able to update the firmware to D2.12.2, however when I place the radio into DFU mode and upgrade the new bootloader nothing happens (it does not update). I've followed the instructions posted on df8oe and am not using the upload function on dfuse.  I've done this process about 4x with the same result and while the radio is working, it does boot to an error stating to update the bootloader, then continues to the radio interface. 
I'm using win10 and have the driver working and recognized.  I had to rename the firmware to match the radio name prior to it updating and have tried the same with the bootlader, with no luck. 

Any help or advice is appreciated.

**Comment by df8oe** [comment, marked as answer]
Bootloader only can be updated with "P6-method". You have to set the jumper on UI board and regarding to the specific hardware you have to press and hold BAND+-Button while powering up (I hope I remeber correct BAND+ and not BAND- - please read the WIKI). Now you can use dfu tool to flash the bootloader.dfu (!!!)

**Reply by buzzkillg230rc** [reply]
Thank you, it took a few tries but eventually got the firmware loaded. 
Cheers KG6YGD
